File: scala/src/main/scala/com/vdzon/administratie/checkandfix/CheckAndFixData.scala
package com.vdzon.administratie.checkandfix

import com.vdzon.administratie.model.boekingen.Boeking
import com.vdzon.administratie.model.{Afschrift, BoekingenCache, Factuur, Rekening}
import scala.collection.JavaConversions._
import scala.compat.java8.StreamConverters._

case class CheckAndFixData(
                  alleAfschriften: List[Afschrift],
                  alleRekeningen: List[Rekening],
                  alleFacturen: List[Factuur],
                  alleBoekingen: List[Boeking],
                  boekingenCache: BoekingenCache)

File: checkandfix/impl/src/main/java/com/vdzon/administratie/checkandfix/CheckAndFixData.kt
package com.vdzon.administratie.checkandfix

import com.vdzon.administratie.model.Afschrift
import com.vdzon.administratie.model.BoekingenCache
import com.vdzon.administratie.model.Factuur
import com.vdzon.administratie.model.Rekening
import com.vdzon.administratie.model.boekingen.Boeking

data class CheckAndFixData(
        val alleAfschriften: List<Afschrift>,
        val alleRekeningen: List<Rekening>,
        val alleFacturen: List<Factuur>,
        val alleBoekingen: List<Boeking>,
        val boekingenCache: BoekingenCache)